WFInfo/Ocr.cs

```diff
@@ -218,7 +212,8 @@ internal static void ProcessRewardScreen(Bitmap file = null)
 
             if (firstChecks == null || firstChecks.Length == 0 || CheckIfError())
             {
-                Main.AddLog(("----  Partial Processing Time, couldn't find rewards " + (Main.watch.ElapsedMilliseconds - start) + " ms  ------------------------------------------------------------------------------------------").Substring(0, 108));
+	            processingActive = false;
+	            Main.AddLog(("----  Partial Processing Time, couldn't find rewards " + (Main.watch.ElapsedMilliseconds - start) + " ms  ------------------------------------------------------------------------------------------").Substring(0, 108));
```

Comment:
Wait, why do we have this substring out of the place?

Reply:
[https://i.imgur.com/Uc8pVCX.png](Here)